packages/ses/README.md

```diff
@@ -392,17 +455,25 @@ const odd = new Compartment({}, {}, {
 });
 ```
 
-### importNowHook
+#### importNowHook
 
-Additionally, an `importNowHook` may be provided that the compartment will use as means to synchronously load modules not seen before in situations where calling out to asynchronous `importHook` is not possible.
-Specifically, when `compartmentInstance.importNow('specifier')` is called, the compartment will first look up module records it's already aware of and call `moduleMapHook` and if none of that is successful in finding a module record matching the specifier, it will call `importNowHook` expecting to synchronously receive the same record type as from `importHook` or throw if it cannot.
+Additionally, an `importNowHook` may be provided that the compartment will use
+as means to synchronously load modules not seen before in situations where
+calling out to asynchronous `importHook` is not possible.
+Specifically, when `compartmentInstance.importNow('specifier')` is called, the
+compartment will first look up module records it's already aware of and call
+`moduleMapHook` and if none of that is successful in finding a module record
+matching the specifier, it will call `importNowHook` expecting to synchronously
+receive the same record type as from `importHook` or throw if it cannot.
 
 ```js
 import 'ses';
-import { StaticModuleRecord } from '@endo/static-module-record';
+import { StaticModuleRecord as ModuleSource } from '@endo/static-module-record';
 
-const c1 = new Compartment({}, {
-  'c2': c2.module('./main.js'),
+const compartment = new Compartment({}, {
+  c: {
+    source: new StaticModuleRecord(''),
```

Comment:
```suggestion
    source: new ModuleSource(''),
```

Reply:
please to a global search for stale names

Reply:
I’ll do a follow-up.